app/components/participation-list/modal.hbs

```diff
@@ -5,6 +5,14 @@
         as |Modal|>
   <Modal.Body>
     <div class="au-o-flow">
+      {{#if this.error}}
```

Comment:
I would put this error next to the "insert" button

atm when you are scrolled down, you don't see the error and it feels like the insert button just doesnt work